src/wp-includes/rest-api/endpoints/class-wp-rest-attachments-controller.php

```diff
@@ -722,7 +721,7 @@ public function edit_media_item( $request ) {
 		$new_image_meta = wp_generate_attachment_metadata( $new_attachment_id, $saved['path'] );
 
 		// Copy the EXIF metadata from the original attachment if not generated for the edited image.
-		if ( isset( $image_meta['image_meta'] ) && isset( $new_image_meta['image_meta'] ) && is_array( $new_image_meta['image_meta'] ) ) {
+		if ( isset( $image_meta['image_meta'], $new_image_meta['image_meta'] ) && is_array( $new_image_meta['image_meta'] ) ) {
```

Comment:
Code didn't use `isset` for two different variable.

Reply:
Hi @mukeshpanchal27, thank you for your review!

> Core didn't use `isset` for two different variables.

After double-checking, it looks like Core does sometimes use a single `isset()` statement for multiple variables. For example, you can find this approach in:
- `WP_Plugin_Install_List_Table::order_callback()`
- `WP_REST_Menu_Items_Controller::prepare_items_query`
- `WP_Community_Events::coordinates_match()`
- parts of the Media Library code and other places.

I thought using a single `isset()` here could be a small optimization since it would prevent calling `isset()` multiple times. What do you think?

Reply:
imho using a single `isset` (as well as `unset`) is fine. I wouldn't go that far calling it a performance optimization, but in my eyes it improves readability.

Reply:
It's not a blocker if we go with a single isset. It's used in some places already, so there are no objections.